students/NatalieRodriguez/Lesson04/dict_lab.py

```diff
@@ -0,0 +1,49 @@
+#$ chmod +x dict_lab.py
+
+#Lesson 04: Dictionary Labs
+#Natalie Rodriguez
+# March 28, 2018
+
+#Dictionaries1
+
+d = {} #created a blank dictionary to populate
+d['Name'] = 'Chris'
+d['City'] = 'Seattle'
+d['Cake'] = 'Chocolate'
+
+print(d)
+
+d.pop('Cake')
+print(d)
+
+d['Fruit'] = 'Mango'
+print(d)
+
+for key, value in d.items() :
+    print (key)
+
+for key, value in d.items() :
+    print (value)
+
+if 'Cake' == key:
+    print(True)
+else: print("There is no cake in this dictionary.")
+
+if 'Mango' == value:
+    print(True)
+else: print("there are no mangos in this dictionary.")
```

Comment:
why are the above comparing to `key` and `value` ? those will hold the last values from the for loop.